packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -245,7 +245,7 @@ export async function copyProjectTemplateAndReplace(
 
   const templateVars: Record<string, any> = {
     useMustache: true,
-    regExpPatternsToRemove: ['//\\sclang-format\\s(on|off)\\s'],
+    regExpPatternsToRemove: ['//\\sclang-format\\s(on|off)\\s*'], // CODESYNC: vnext\local-cli\runWindows\utils\autolink.js
```

Comment:
You could also just disable clang-format in the whole aggregate template directory by nesting a config file for it, instead having the template file disable it. We used to do that for C++ overrides.